showcase/src/app/demo/index.ts

```diff
@@ -2,3 +2,6 @@ export * from "./action-bar";
 export * from "./example-viewer";
 export * from "./table";
 export * from "./demo.module";
+export * from "./action-bar/action-bar.component";
+export * from "./button/button.component";
```

Comment:
This barrel should just re-export everything from `./action-bar`. There, inside the "action-bar" folder there should be a barrel `index.ts` which re-exports all the `.ts` files except the `spec.ts`.

See how it is done in the other folders ;)